src/Sulu/Bundle/AdminBundle/Resources/js/components/SingleIconSelect/SingleIcon.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import singleIconSelectStyle from './singleIconSelect.scss';
+
+type Props = {
+    content: string,
+    id: string | number,
+    isSelected: boolean,
+    onClick?: (id: string | number, selected: boolean) => void,
+};
+
+export default class SingleIcon extends React.PureComponent<Props> {
```

Comment:
as far as i see, this component is not used by the `SingleIconSelect` component directly (only indirectly via the `IconAdapter`). 

therefore i would prefer to make this consistent to the `MediaCardAdapter` and call it `IconCard` and store it in a separate folder (`components/IconCard`)

Reply:
also, we should add basic test cases for the component before merging it